tests/bsim/bluetooth/host/att/mtu_update/src/main.c

```diff
@@ -39,59 +39,85 @@ extern void run_peripheral_sample(uint8_t *notify_data, size_t notify_data_size,
 
 extern enum bst_result_t bst_result;
 
-#define CREATE_FLAG(flag) static atomic_t flag = (atomic_t)false
-#define SET_FLAG(flag) (void)atomic_set(&flag, (atomic_t)true)
-#define WAIT_FOR_FLAG(flag)                                                                        \
-	while (!(bool)atomic_get(&flag)) {                                                         \
-		(void)k_sleep(K_MSEC(1));                                                          \
-	}
-
 #define WAIT_TIME (20e6) /* 20 seconds */
-#define PERIPHERAL_NOTIFY_TIME ((WAIT_TIME - 10e6) / 1e6)
+#define PERIPHERAL_WAIT_TIME ((WAIT_TIME - 10e6) / 1e6)
 
-CREATE_FLAG(flag_notification_received);
-uint8_t notify_data[100] = {};
-uint8_t is_data_equal;
```

Comment:
Does it make sense to remove the notification part of the test? 

Wouldn't it make sense to both verify the MTU values **and** verify the notification length?